src/Data/Acid/Archive.hs

```diff
@@ -18,10 +18,11 @@ import           Data.Acid.CRC
 
 import qualified Data.ByteString        as Strict
 import qualified Data.ByteString.Lazy   as Lazy
+import           Data.ByteString.Lazy.Builder
```

Comment:
Btw, this would become Data.ByteString.Builder if you want to raise the bytestring dependency a bit. The Data.ByteString.Lazy.Builder is only provided for compatibility right now and deprecated